index.ts

```diff
@@ -582,7 +582,9 @@ export interface IlpRejection {
   code: string,
   triggeredBy: string,
   message: string,
-  data: Buffer
+  data: Buffer,
+  receivedAmount?: string,
+  maximumAmount?: string
```

Comment:
I think the better way of doing this with Typescript would be to have a type that is an F08 error, rather than adding optional fields to all rejections.